linux/net/rina/rmt-ps-common.c

```diff
@@ -0,0 +1,217 @@
+/*
+ * Common parts of the common policy set for RMT.
+ * This is envisioned for reurilization in other custom policy sets
+ *
+ *    Leonardo Bergesio <leonardo.bergesio@i2cat.net>
```

Comment:
Please keep the authors of the previous rmt-ps-default.c implementation (v.maffione@nextworks.it)